### PR TITLE
move install script to the top

### PR DESCRIPTION
**Pull request by lawndoc**
Related to issue #10489 that I just opened.

The installer script will be the most relevant for people visiting this page. It makes sense to make it more visible at the top of the page.

I almost missed it when I was reading the documentation because of its previous placement.

### CODE REVIEW

**Comment by lawndoc** [comment]
> @lawndoc please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Vermeer Corporation"]

**Comment by lawndoc** [comment]
@microsoft-github-policy-service agree company="Vermeer Corporation"

File: microsoft-365/security/defender-endpoint/linux-install-manually.md [hunk 1]
```diff
@@ -53,7 +53,29 @@ Before you get started, see [Microsoft Defender for Endpoint on Linux](microsoft
 > [!WARNING]
 > Upgrading your operating system to a new major version after the product installation requires the product to be reinstalled. You need to [Uninstall](linux-resources.md#uninstall-defender-for-endpoint-on-linux) the existing Defender for Endpoint on Linux, upgrade the operating system, and then reconfigure Defender for Endpoint on Linux following the below steps.
 
+## Installer script
 
+The easiest way to deploy Defender for Endpoint on Linux is with an automated [installer bash script](https://github.com/microsoft/mdatp-xplat/blob/master/linux/installation/mde_installer.sh) provided in our [public GitHub repository](https://github.com/microsoft/mdatp-xplat/).
+The script identifies the distribution and version, simplifies the selection of the right repository, sets up the device to pull the latest package, and combines the product installation and onboarding steps. Details on the steps performed can be found in the manual set up instructions below this section.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-install-manually.md]:
```suggestion
The script identifies the distribution and version, simplifies the selection of the right repository, sets up the device to pull the latest package, and combines the product installation and onboarding steps. Details on the steps to be performed are provided in the manual setup instructions following this section.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lawndoc, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@lawndoc Gentle reminder.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by siosulli** [comment]
@lawndoc @yogkumgit @JohanFreelancer9 

Thanks so much for your input! I have spoken to the product manager about this feedback and we won't move this content for now.

This is for two reasons: 

1. This scripted install (which is a different script from the onboarding script) is not preferred over other methods – it’s intended for those scenarios where deployment tooling is not available. 
2. We plan to split this topics up into a few different topics - once we do, we will make sure this script is in the right place. 

Thanks again for your feedback and I have tracked it to make sure we implement it the reworked content.

**Comment by lawndoc** [comment]
> This scripted install (which is a different script from the onboarding script) is not preferred over other methods – it’s intended for those scenarios where deployment tooling is not available.

You can't move the script up on the page dedicated to a manual install? It's not like it would be moved above the preferred deployment tooling that you mentioned, it's just letting people skip a bunch of reading when they don't have deployment tooling and they have found themself on this page.

**Comment by lawndoc** [comment]
This is why Microsoft documentation sucks, I'm done trying to help

**Pull request closed without merging**